apisix/init.lua

```diff
@@ -620,6 +619,15 @@ end
 
 function _M.http_log_phase()
     local api_ctx = common_phase("log")
+    if not api_ctx then
+        local ctx = ctxdump.apply_ngx_ctx(ngx_var.ctx_ref)
```

Comment:
Why not reapply the ctx in `http_header_filter_phase`?

Reply:
Good. Let me fix it.

Reply:
Fixed. Please review it again.